guides/image-sources/provisioning-installer-bootdisk-fullhost.plantuml

```diff
@@ -0,0 +1,73 @@
+@startuml
+
+!include foreman.pstyle
+!$networkboot = 0
+
+title Installer-based provisioning with full-host boot disk on bare metal
+
+left header
+In the described scenario, Foreman does not manage DHCP.
```

Comment:
Where is this header displayed? I don't see it in the picture

Reply:
Top left corner. Unfortunately, the font is too small. I'll try to find out if I can make it larger.